micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/metrics/config/MetricsConfiguration.groovy

```diff
@@ -68,4 +70,11 @@ class MetricsConfiguration {
     MetricRegistry metricRegistry(MetricPathProvider metricPathProvider) {
         return new PathPrependingMetricRegistry(metricPathProvider)
     }
+
+    @Bean
+    public HystrixCodaHaleMetricsPublisher hystrixCodaHaleMetricsPublisher(MetricRegistry metricRegistry) {
+        def publisher = new HystrixCodaHaleMetricsPublisher(metricRegistry)
```

Comment:
At least I know what type is there that should be registered :) But of course I'm not an antidef evangelist ;)